test/src/test/java/org/corfudb/integration/WorkflowIT.java

```diff
@@ -730,4 +753,736 @@ public void testRuntimeGCWithStreamWithNoUpdatesAfterCheckpoint() throws Excepti
+                });
+                runtimeIndex++;
+            }
+
+            executor.shutdown();
+            executor.awaitTermination(2, TimeUnit.MINUTES);
+            System.out.println(String.format("**** Multi-threaded puts (%s threads, %s keys) completed in: %s ms",
+                    numThreads, numKeys, (System.currentTimeMillis() - startTime)));
+
+            // Read from fresh runtime
+            CorfuTable<Integer, String> table2 = rtReader.getObjectsView().build()
+                    .setTypeToken(new TypeToken<CorfuTable<Integer, String>>() {
+                    })
+                    .setStreamName("streamTable")
+                    .open();
+
+            startTime = System.currentTimeMillis();
+            assertThat(table2.size()).isEqualTo(numKeys);
+            System.out.println(String.format("**** New runtime read completed in: %s ms", (System.currentTimeMillis() - startTime)));
+        } catch(Exception e) {
+            // Exception
+        } finally {
+            for(CorfuRuntime rt : runtimes) {
+                rt.shutdown();
+            }
+            rtReader.shutdown();
+            shutdownCorfuServer(server);
+        }
+    }
+
+    @Test
+    @Ignore
+    public void testCacheWithNulls() throws Exception {
+        final int cacheSize = 10;
+        final long address2 = 2L;
+        final long address3 = 3L;
+        final long address4 = 4L;
+        final long address5 = 5L;
+        final LoadingCache<Long, ILogData> readCache = CacheBuilder.newBuilder()
+                .maximumSize(cacheSize)
+                .build(new CacheLoader<Long, ILogData>() {
+                    @Override
+                    public ILogData load(Long value) throws Exception {
+                        return null;
+                    }
+
+                    @Override
+                    public Map<Long, ILogData> loadAll(Iterable<? extends Long> keys) throws Exception {
+                        System.out.println("**** Load all: " + keys);
+                        Map<Long, ILogData> mapReturn = new HashMap<>();
+                        mapReturn.put(address2, new LogData(DataType.EMPTY));
+                        mapReturn.put(address3, new LogData(DataType.EMPTY));
+                        mapReturn.put(address4, new LogData(DataType.EMPTY));
+                        mapReturn.put(address5, null);
+
+                        //                        keys.forEach(address -> mapReturn.put(address, null));
+                        return mapReturn;
+                    }
+                });
+
+        final Cache<Long, ILogData> writeCache = CacheBuilder.newBuilder()
+                .maximumSize(cacheSize)
+                .recordStats()
+                .build();
+
+        // Check if null's can be set directly
+        //readCache.put(0L, null);
+        //writeCache.put(0L, null);
+//        writeCache.put(6L, null);
+//        readCache.put(2L, new LogData(DataType.EMPTY));
```

Comment:
Can these commented lines be removed?

Reply:
Oh, actually I removed this test as it was something experimental for the loading cache!